third_party/proto/cosmos/authz/v1beta1/authz.proto

```diff
@@ -19,6 +19,18 @@ message GenericAuthorization {
   string msg = 1;
 }
 
+// CountAuthorization gives the grantee unrestricted permissions to execute
```

Comment:
i thought CountAuthorization was coming via 0.46 or i guess it was back ported into 0.45?  just making sure we are good

Reply:
It's the opposite. It's in 0.45 and will be back-ported in 0.46 (since our goal is to launch 1.12 with 0.46).